dleq/dleq.go

```diff
@@ -1,3 +1,6 @@
+// Package dleq provides and interface named Interface layer on top of the cgo-dleq layer
+// that in turn provides a golang interface to the Distributed Log Equality (DLEQ)
+// algorithm code written in Rust.
```

Comment:
I definitely need help on this one.

Reply:
// Package dleq provides the sub-api that we need from the cgo-dleq package, which in turn is a wrapper to the Discrete Log Equality (DLEQ) algorithm code written in Rust.